test/compress/harmony.js

```diff
@@ -183,8 +183,9 @@ import_statement: {
         import { Bar, Baz } from 'lel';
         import Bar, { Foo } from 'lel';
         import { Bar as kex, Baz as food } from 'lel';
+        import * from 'lel';
     }
-    expect_exact: "import\"mod-name\";import Foo from\"bar\";import{Bar,Baz}from\"lel\";import Bar,{Foo}from\"lel\";import{Bar as kex,Baz as food}from\"lel\";"
+    expect_exact: "import\"mod-name\";import Foo from\"bar\";import{Bar,Baz}from\"lel\";import Bar,{Foo}from\"lel\";import{Bar as kex,Baz as food}from\"lel\";import*from\"lel\";"
```

Comment:
As I was just adding into an existing test, I kept the form of the test as it was before. I prefer not mixing unrelated cleanup / refactoring with changes, but if you want, I can certainly change the quotes.